oap-server/server-configuration/configuration-etcd/pom.xml

```diff
@@ -39,10 +39,76 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.apache.skywalking</groupId>
-            <artifactId>cluster-etcd-plugin</artifactId>
-            <version>${project.version}</version>
+            <groupId>org.projectlombok</groupId>
```

Comment:
etcd4j is existing in dependency management. You should not add it here.

Reply:
ok

Reply:
here, i should add the dependency, or it will compile failed.

Reply:
@wayilau he means the `projectlombok`, it's included in `oap-server/pom.xml` already am I right @wu-sheng

Reply:
@wayilau can you try to remove the `projectlombok` dependency? I think it's included in the `oap-server/pom.xml`, other codes look good to me